tide_core.install

```diff
@@ -378,3 +378,22 @@ function tide_core_update_10012() {
   $shield_config = \Drupal::configFactory()->getEditable('shield.settings');
   $shield_config->set('paths', $paths_string)->save();
 }
+
+/**
+ * Add custom filters field storage.
+ */
+function tide_core_update_10013() {
```

Comment:
Please make sure this update hook runs before the others in this PR. You can refer to this for guidance https://api.drupal.org/api/drupal/core!lib!Drupal!Core!Extension!module.api.php/function/hook_update_dependencies/10